File: pythonGemMigration/pipelines/Pipeline_first/code/pipeline_first/graph/target_5.py
from pyspark.sql import *
from pyspark.sql.functions import *
from pyspark.sql.types import *
from prophecy.utils import *
from prophecy.libs import typed_lit
from pipeline_first.config.ConfigStore import *
from pipeline_first.udfs.UDFs import *

def target_5(spark: SparkSession, in0: DataFrame):
    in0.write\
        .format("jdbc")\
        .option("url", "jdbc:mysql://3.101.152.38:3306/test_database")\
        .option("dbtable", "test_table_destination")\
        .option("user", "test_user")\
        .option("password", "admin")\
        .option("driver", "com.mysql.jdbc.Driver")\
        .mode("overwrite")\
        .save()

File: GemMigrationScala/pipelines/FirstPipeline/code/src/main/scala/io/prophecy/pipelines/firstpipeline/graph/target_5.scala
package io.prophecy.pipelines.firstpipeline.graph

import io.prophecy.libs._
import io.prophecy.pipelines.firstpipeline.config.Context
import org.apache.spark._
import org.apache.spark.sql._
import org.apache.spark.sql.functions._
import org.apache.spark.sql.types._
import org.apache.spark.sql.expressions._
import java.time._

object target_5 {

  def apply(context: Context, in: DataFrame): Unit = {
    val Config = context.config
    var writer = in.write.format("jdbc")
    writer = writer
      .option("url",      s"${Config.JDBC}${Config.JDBC_DATABASE}")
      .option("dbtable",  s"${Config.TEST}_table_destination")
      .option("user",     s"${Config.TEST}_${Config.USER}")
      .option("password", s"${Config.ADM}${Config.IN}")
      .option("driver",   "com.mysql.jdbc.Driver")
    writer = writer.mode("overwrite")
    writer.save()
  }

}
